### ISSUE TITLE
Bash commands are not executed

### ISSUE THREAD

**Issue by cheap-glitch** [Ulauncher/Ulauncher#1006]
### Environment

Arch Linux + Wayland

### Bug description

Typing a command in Ulauncher and selecting the "Execute" option does nothing. This is the tail output of `systemctl --user status ulauncher`:

```
mars 31 08:20:47 HAL9000 ulauncher[8810]: bash: cannot set terminal process group (1387): Inappropriate ioctl for device
mars 31 08:20:47 HAL9000 ulauncher[8810]: bash: no job control in this shell
mars 31 08:20:47 HAL9000 ulauncher[8810]: bash: None: command not found
```

### Log output

```shell
Ulauncher version 6.0.0
Extension API version 2.0
GTK+ 3.24.33
Session type: wayland
X11 backend: Yes
```

### Communication guidelines

- [X] I have read and agree with Ulauncher's communication guidelines and checked the troubleshooting page for self-help.

### Not an extension issue

- [X] This issue applies to the Ulauncher application, not a specific extension.

### Latest version

- [X] I am using the latest Ulauncher release.

**Comment by friday** [comment]
This one is harder to fix I think. I don't even know what is causing this based on the error message. You can disable the systemd job and run ‘"ulauncher" as a command from your terminal or something in the meanwhile maybe. Sorry for the inconvenience, and thanks for helping us test 👍

**Comment by friday** [comment]
It's trying to run "None" as a command, but there is no such command, and it shouldn't try to run it.

**Comment by friday** [comment]
"ulauncher --no-extensions -v" could possibly avoid the error but it disables extensions, so it's not a real solution. Just something to test possibly.

**Comment by friday** [comment]
I noticed I didn't read this properly before. It is isolated to running a shortcut with an argument?

**Comment by cheap-glitch** [comment]
> It is isolated to running a shortcut with an argument?

No, I get the same errors whether the command is just an executable name, or is a name with some arguments.

**Comment by friday** [comment]
But it's still shortcuts? Not apps or extensions?

**Comment by cheap-glitch** [comment]
> But it's still shortcuts? Not apps or extensions?

Just to be perfectly clear, I'm talking about the "Execute" option:

![image](https://user-images.githubusercontent.com/46634000/162137042-a27fc152-6ef2-4c68-b8e0-ffc4023b7a9d.png)

**Comment by friday** [comment]
That's not a Ulauncher feature. It's an extension or a shortcut

**Comment by friday** [comment]
Looks like a shortcut. Meaning it's your own code or code you copied from someone else. And you enabled it as a "default search". Pretty smart though, but it's not Ulauncher's code :+1:

**Comment by friday** [comment]
Will reopen if you post the shortcut code and settings and we find that the code doesn't work and that it should :)

**Comment by cheap-glitch** [comment]
> Looks like a shortcut. And you enabled it as a "default search"

Right, completely forgot I did that :facepalm: 

```json
"f18e64c0-df26-4009-a996-9e2e55d6c294": {
	"id": "f18e64c0-df26-4009-a996-9e2e55d6c294",
	"name": "Execute",
	"keyword": "!",
	"cmd": "#!/usr/bin/env bash\nbash -c \"${*}\"",
	"icon": "~/.config/ulauncher/icons/bash.png",
	"is_default_search": true,
	"run_without_argument": false,
	"added": 1640718026.1338382
}
```

I had a `-i` in the `cmd` which caused the ioctl error. But I still get this one:

```
avril 09 08:39:57 HAL9000 ulauncher[3579]: bash: line 1: None: command not found
```

It seems like the arguments are not properly passed to the command?

**Comment by friday** [comment]
The argument should be named $1, not ${*}

**Comment by friday** [comment]
$s should work. It's a simple string replace.

https://github.com/Ulauncher/Ulauncher/blob/1b66c7636ae8614ab880db7eda6b3e5d718a3fbc/ulauncher/modes/shortcuts/ShortcutResult.py#L57

However I just found this comment, indicating %s was actually never intended to work for the shell arguments while actual shell argument code did:
https://github.com/Ulauncher/Ulauncher/blob/fe9f613a8764cd0ef2a8d1fe6ab205895e460b2a/preferences-src/src/components/pages/EditShortcut.vue#L33

So this is either a regression, or breaking change. I see no reason to remove support for shell arguments though, so I'll try to get those working also. You can use $s in the meanwhile.

**Comment by friday** [comment]
Fixed in aabe254b77f7db88b5180168eeb698f1d0f6b7e4

You can also see the stdout from shortcut scripts now if you run ulauncher in verbose mode (`-v` cli flag) which should help debugging them c0eff5e1b7ae50d5697bc41c569adccb8a7dffa1
